modules/local/consolidatebcfs/main.nf

```diff
@@ -0,0 +1,37 @@
+/*
+This source file is adapted from a nf-core pipeline for SNVPhyl developed by
+Jill Hagey as a work of the United States Government, which was distributed as
+a work within the public domain under the Apache Software License version 2.
+
+This source file has been adapted to work within our pipeline.
+
+Please refer to the README for more information.
+*/
+process CONSOLIDATE_BCFS {
+    tag "${meta.id}"
+    label 'process_single'
+    container = "staphb/snvphyl-tools:1.8.2"
+
+    input:
+    tuple val(meta), path(mpileup_bcf), path(freebayes_filtered_bcf), path(freebayes_filtered_csi)
+
+    output:
+    path( "${meta.id}_consolidated.bcf" ),     emit: consolidated_bcfs
+    path( "${meta.id}_consolidated.vcf" ),     emit: consolidated_vcfs
+    path( "${meta.id}_consolidated.bcf.csi" ), emit: consolidated_bcf_index
+    path( "${meta.id}_filtered_density.txt" ), emit: filtered_densities
+    path("versions.yml"),                      emit: versions
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    consolidate_vcfs.pl --coverage-cutoff 5 --min-mean-mapping 30 --snv-abundance-ratio 0.75 --vcfsplit ${freebayes_filtered_bcf} --mpileup ${mpileup_bcf} --filtered-density-out ${prefix}_filtered_density.txt --window-size ${params.window_size} --density-threshold ${params.density_threshold} -o ${prefix}_consolidated.bcf > ${prefix}_consolidated.vcf
+    bcftools index -f ${prefix}_consolidated.bcf
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        snvphyl-tools:
```

Comment:
Can you add the string `1.8.2` here so it reports the version of snvphyl-tools. We just have to remember to change it if the container changes.

Reply:
Added in e8252e6847eeb0aad4caa3cbf1ba760bf2c465ed

Reply:
Thanks. Could you add the version it to all the other tools with `snvphyl-tools` as well?

Reply:
Added in 220e2acfa84260dd11bf767cbdab52a5c764d015